qutebrowser/browser/downloads.py

```diff
@@ -239,6 +239,7 @@ class FileDownloadTarget(_DownloadTarget):
     def __init__(self, filename):
         # pylint: disable=super-init-not-called
         self.filename = filename
+        # pylint: enable=super-init-not-called
```

Comment:
No need for those with `super-init-not-called`, as pylint already only turns things off for this function and it's needed for the entire function.